### PR TITLE
split coverage and ci. Add tarpaulin args

### PR DESCRIPTION
**Pull request by jdroenner**

### CODE REVIEW

**Comment by jdroenner** [comment]
[Split CI and Coverage. Fix Coverage.](https://trello.com/c/SCGtk1a3/989-split-ci-and-coverage-fix-coverage)

**Review by ChristianBeilschmidt — APPROVED** [review]

**Comment by ChristianBeilschmidt** [comment]
bors merge

**Comment by ChristianBeilschmidt** [comment]
Can you increase the timeout for bors?
https://forum.bors.tech/t/bors-keeps-timing-out/87

**Review by ChristianBeilschmidt — DISMISSED** [review]

**Comment by ChristianBeilschmidt** [comment]
bors merge

**Comment by ChristianBeilschmidt** [comment]
bors merge

**Review by ChristianBeilschmidt — APPROVED** [review]

**Comment by ChristianBeilschmidt** [comment]
bors merge

File: bors.toml [hunk 1]
```diff
@@ -14,3 +14,6 @@ status = [
 
 # One approval necessary for bors to work
 required_approvals = 1
+
+# Set bors's timeout to 4 hours
+timeout-sec = 14400
```
Comment (ChristianBeilschmidt) [inline bors.toml:19]: 4 hours is quite long, what about 2 hours?

**Review by ChristianBeilschmidt — CHANGES_REQUESTED** [review]
You need to change the https://github.com/geo-engine/geoengine/blob/master/bors.toml to reflect the new name of the coverage task.
I think it is now `coveralls` instead of `coverage`.
This is why we cannot merge it.

**Review by ChristianBeilschmidt — APPROVED** [review]

**Comment by ChristianBeilschmidt** [comment]
bors merge

**Pull request merged**